### PR TITLE
frontend/fix-added estimates stage

### PR DESCRIPTION
**Pull request by MercyQueen**
- Added estimates stage in between rentalScreen and quotes Screen
https://github.com/user-attachments/assets/018a6c97-d0bc-4f6d-a913-0b015f17360e

### CODE REVIEW

**Review by aryan-13 — APPROVED** [review]

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/ComponentConfig.purs [hunk 1]
```diff
@@ -471,6 +472,20 @@ rentalRateCardConfig state =
...
+
+    autoAnimConfig' =
+      config
+        { duration = 400
+        , toScaleX = 1.0
+        , toScaleY = 1.0
+        }
+  in
+    autoAnimConfig'
+
```
Comment (spravinkumar9952) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/ComponentConfig.purs:488]: here let and in is not needed. we can directly override the values

autoAnimConfig :: AnimConfig.AnimConfig
autoAnimConfig = AnimConfig.animConfig
        { duration = 400
        , toScaleX = 1.0
        , toScaleY = 1.0
        }

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs [hunk 2]
```diff
@@ -775,6 +777,84 @@ saveFavCardView push state globalProps =
...
+          , gravity CENTER
+          , background Color.white900
+          ][ imageView
+              [ imageWithFallback $ fetchImage FF_ASSET "ny_ic_wallet_filled"
+              , height $ V 20
+              , width $ V 20
+              , margin $ MarginTop 3
+              ]
+            , textView $
+              [ text (getString PAY_DRIVER_USING_CASH_OR_UPI)
```
Comment (spravinkumar9952) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs:806]: $

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs [hunk 3]
```diff
@@ -775,6 +777,84 @@ saveFavCardView push state globalProps =
...
+    , padding $ PaddingVertical 40 24 
+    , background Color.white900
+    , stroke $ "1," <> Color.grey900
+    , cornerRadii $ Corners 24.0 true true false false
+    , clickable true
+    , gravity CENTER_HORIZONTAL
+    ][
+      PrestoAnim.animationSet [ scaleAnim $ autoAnimConfig ]
+        $ lottieAnimationView
+    [ id (EHC.getNewIDWithTag "lottieLoaderSearchLocationScreen")
```
Comment (spravinkumar9952) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs:835]: $

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs [hunk 4]
```diff
@@ -775,6 +777,84 @@ saveFavCardView push state globalProps =
...
+              , margin $ MarginTop 3
+              ]
+            , textView $
+              [ text (getString PAY_DRIVER_USING_CASH_OR_UPI)
+              , accessibilityHint "Pay Driver using Cash/UPI : Text"
+              , accessibility ENABLE
+              , lineHeight "18"
+              , width WRAP_CONTENT
+              , height WRAP_CONTENT
+              , padding (Padding 5 20 0 16)
```
Comment (spravinkumar9952) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs:812]: $

File: Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs [hunk 5]
```diff
@@ -775,6 +777,84 @@ saveFavCardView push state globalProps =
...
+    , stroke $ "1," <> Color.grey900
+    , cornerRadii $ Corners 24.0 true true false false
+    , clickable true
+    , gravity CENTER_HORIZONTAL
+    ][
+      PrestoAnim.animationSet [ scaleAnim $ autoAnimConfig ]
+        $ lottieAnimationView
+    [ id (EHC.getNewIDWithTag "lottieLoaderSearchLocationScreen")
+    , afterRender
+        ( \action -> do
```
Comment (spravinkumar9952) [inline Frontend/ui-customer/src/Screens/SearchLocationFlow/SearchLocationScreen/View.purs:837]: can we shadow the action. also the do and pure unit are not necessary.

**Review by spravinkumar9952 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**